support/monitoring/README.md

```diff
@@ -0,0 +1,117 @@
+# Setting of Monitoring for JASECI #
+
+
+## Prerequisite ##
+
+### Install Helm  ###
+
+https://helm.sh/docs/intro/install/
+
+
+
+## Promethues ##
+
+#### Step 1 ####
+
+##### add prometheus Helm repo  #####
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+
+#### Step 2 ####
+
+helm install prometheus prometheus-community/prometheus \
+    --set alertmanager.persistentVolume.storageClass="gp2" \
+    --set server.persistentVolume.storageClass="gp2"
+
+The Prometheus server can be accessed via port 80 on the following DNS name from within your cluster:
+prometheus-server.prometheus.svc.cluster.local
+
+#### Step 3 ####
+
+kubectl get all
+
+#### Step 4 ####
```

Comment:
The prometheus pods don't come up successfully following this guide. I am testing this locally with docker-desktop's k8s daemon (i.e. not on any cloud providers).
```
prometheus-alertmanager-8bc89d94-lg5p7           0/2     Pending            0               7m21s
prometheus-kube-state-metrics-6c44ff7fb6-pd8k6   1/1     Running            0               7m21s
prometheus-node-exporter-bp7ds                   0/1     CrashLoopBackOff   6 (92s ago)     7m21s
prometheus-pushgateway-864877484c-mxr4j          1/1     Running            0               7m21s
prometheus-server-788788f7b4-xkndl               0/2     Pending            0               7m21s
```

Reply:
describe the prometheus-server pod gives the following
```
Events:
  Type     Reason            Age    From               Message
  ----     ------            ----   ----               -------
  Warning  FailedScheduling  8m46s  default-scheduler  0/1 nodes are available: 1 pod has unbound immediate PersistentVolumeClaims.
  Warning  FailedScheduling  7m16s  default-scheduler  0/1 nodes are available: 1 pod has unbound immediate PersistentVolumeClaims.
```

Reply:
added the details